general-concepts/copyright-policy/text.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
+<guide self="general-concepts/copyright-policy/">
+<chapter>
+<title>Gentoo's copyright policy</title>
+<body>
+
+<p>
+<uri link="https://www.gentoo.org/glep/glep-0076.html">GLEP-76</uri> defines
+copyright and license policies for Gentoo Linux. 
+</p>
+
+<p>
+Every Gentoo project must abide by the
+<uri link="https://www.gentoo.org/glep/glep-0076.html#social-contract">Gentoo
```

Comment:
https://www.gentoo.org/get-started/philosophy/social-contract.html would be a better URI for the Social Contract.